client/app/models/BillingService.class.js

```diff
@@ -110,7 +110,7 @@ export default class BillingService {
   }
 
   setForResiliation() {
-    if (this.hasAutomaticRenew() && !this.isAutomaticallyRenewed()) {
+    if (this.hasAutomaticRenew() && !(this.isAutomaticallyRenewed() || ['automaticV2014', 'automaticV2016'].includes(this.renewalType))) {
```

Comment:
shouldn't `['automaticV2014', 'automaticV2016'].includes(this.renewalType)` be part of the `hasAutomaticRenew` method? i find those test confusing but maybe you don't a have better choice :/

Reply:
No because you can have automatic renew but have different type/version